kvbc/include/categorization/shared_kv_category.h

```diff
@@ -0,0 +1,89 @@
+// Concord
+//
+// Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the
+// "License").  You may not use this product except in compliance with the
+// Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+
+#pragma once
+
+#include "kv_types.hpp"
+#include "rocksdb/native_client.h"
+
+#include "base_types.h"
+#include "categorized_kvbc_msgs.cmf.hpp"
+
+#include <memory>
+#include <string>
+
+namespace concord::kvbc::categorization::detail {
+
+// The shared key-value category persists key-values across multiple categories. A category is implemented as a pair of
+// RocksDB column families:
+//  * `data` column family - contains versioned key-values for the given category
+//  * `latest version` column family - contains the latest version of a key for the given category
```

Comment:
maybe add a sentence on the usage of this column family

Reply:
Good call, doing that.